### PR TITLE
Adding Primitive POS with minimal features

### PR DESCRIPTION
**Pull request by nurdtechie98**
Moved the initial PR #41 from move-to-vue to master as instructed by @surajshetty3416 will append the remaining features developed in some time

### CODE REVIEW

File: models/doctype/Event/Event.js [hunk 1]
```diff
@@ -35,7 +35,12 @@ module.exports = {
       fieldtype: 'Table',
       childtype: 'EventSchedule',
       label: 'Schedule'
-    }
+    },
+    {
+      fieldname: "eventposter",
```
Comment (netchampfaris) [inline models/doctype/Event/Event.js]: is this required?

File: models/doctype/Item/Item.js [hunk 2]
```diff
@@ -54,6 +54,11 @@ module.exports = {
       fieldname: 'rate',
       label: 'Rate',
       fieldtype: 'Currency'
+    },
+    {
+      fieldname: 'itemImage',
```
Comment (netchampfaris) [inline models/doctype/Item/Item.js]: just call it `image`

File: src/pages/PointOfSale/BillTemplate.vue [hunk 3]
```diff
@@ -0,0 +1,111 @@
+<template>
+	<div class="bill">
```
Comment (netchampfaris) [inline src/pages/PointOfSale/BillTemplate.vue:2]: fix html formatting

File: src/pages/PointOfSale/BillTemplate.vue [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+</template>
+
+<script>
+export default {
+  name: 'bill-template',
+  props: ['items', 'customer', 'netTotal', 'grandTotal'],
+  computed: {
+    today() {
+      // get current date
+      let today = new Date();
```
Comment (netchampfaris) [inline src/pages/PointOfSale/BillTemplate.vue]: use `luxon` for Date operations

File: src/pages/PointOfSale/Billing.vue [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+      if (dd < 10) {
+        dd = "0" + dd;
+      }
+      if (mm < 10) {
+        mm = "0" + mm;
+      }
+      today = mm + "/" + dd + "/" + yyyy;
+
+      // create new pdf object
+      let pdf = new jsPDF("p", "mm", [58, 100]);
```
Comment (netchampfaris) [inline src/pages/PointOfSale/Billing.vue]: Make the template using HTML and then convert it to PDF. We already have a `frappe.getPDF` method. Just see how the PDF for Invoice is generated.

File: src/pages/PointOfSale/Checkout.vue [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+    components: {
+      Numpad
+    },
+    props: ["netTotal", "grandTotal", "createInvoice"],
+    data() {
+      return {
+        netTotal: 0,
+        value:"",
+        cash: 0,
+				cheque: 0,
```
Comment (netchampfaris) [inline src/pages/PointOfSale/Checkout.vue]: formatting issues?

**Review by netchampfaris — CHANGES_REQUESTED** [review]
Try to fix all the formatting issues. Also run your code through ESLint to fix all the style issues.

File: models/doctype/Event/Event.js (hunk 1 above)
Comment (nurdtechie98) [reply models/doctype/Event/Event.js]: nope, was just testing the file, forgot to remove.

File: src/pages/PointOfSale/BillTemplate.vue (hunk 3 above)
Comment (karan28598) [reply src/pages/PointOfSale/BillTemplate.vue:2]: Fixed the HTML Formatting and fixed all style issues with ESLint

**Pull request closed without merging**